src/utils/data_types.h

```diff
@@ -14,7 +14,6 @@
 #include <cassert>
 
 #include "global_params.h"
-#include "openmvg_descriptor.hpp"
```

Comment:
Is libvot still able to read openmvg feature without this file? I kind of forget why I add it in the first place.